src/modules/task.js

```diff
@@ -0,0 +1,9 @@
+class Task {
+  constructor(description = 'All clean', completed = false, index = 0) {
```

Comment:
- [ ] Nice to use default values 👌, But I think the `All clean` value it's not necessary and will never be used (YGANI)

Reply:
Thanks for highlighting that :+1: .